flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -106,11 +112,17 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
             return
         }
 
-        PrivateSentrySDKOnly.appStartMeasurementHybridSDKMode = true
-
         SentrySDK.start { options in
             self.updateOptions(arguments: arguments, options: options)
 
+            if let enableAutoPerformanceTracking = arguments["enableAutoPerformanceTracking"] as? Bool,
+               enableAutoPerformanceTracking {
+                PrivateSentrySDKOnly.appStartMeasurementHybridSDKMode = true
+                #if os(iOS)
```

Comment:
We actually need to check if there's UI Kit and not iOS only.
https://github.com/getsentry/sentry-cocoa/blob/17969fa1c80d643df62aa92e8a5fa724c9af0d8c/Sources/Sentry/SentryFramesTrackingIntegration.m#L22-L36

https://github.com/getsentry/sentry-cocoa/blob/f893682911568e14782eb0bb507af5a12332b033/Sources/Sentry/Public/SentryDefines.h#L9-L19

Does it work if we do `os(iOS) || whatElseWOrks`?

Reply:
@philipphofmann Can you help us out here?

Reply:
It should work for:

> This feature is available for iOS, tvOS, and Mac Catalyst.

Reply:
> Does it work if we do os(iOS) || os(whatElseWorks)?

Yes it does

```suggestion
               #if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
```

Reply:
As far as I know Flutter does not work with tvOS.

Reply:
👍 to what @brustolin wrote.

Reply:
Indeed, no tvOS for now, https://docs.flutter.dev/deployment/macos